src/smart-components/role/add-role-permissions/add-role-permission-wizard.js

```diff
@@ -47,10 +46,7 @@ const AddRolePermissionWizard = ({ role }) => {
   const setWizardError = (error) => setWizardContextValue((prev) => ({ ...prev, error }));
   const setWizardSuccess = (success) => setWizardContextValue((prev) => ({ ...prev, success }));
   const setHideForm = (hideForm) => setWizardContextValue((prev) => ({ ...prev, hideForm }));
-
-  useEffect(() => {
-    setSchema(schemaBuilder(container.current));
-  }, []);
+  const schema = useMemo(() => schemaBuilder(container.current));
```

Comment:
You want to add a trigger to the useMemo so it does not computer on each render.

```js
const schema = useMemo(() => schemaBuilder(container.current), []);
```

Reply:
ah, forgot to add that, thank you. Fixed